tools/src/commands/RunStories.ts

```diff
@@ -0,0 +1,241 @@
+    JSON.stringify(mergedPkg, null, '\t')
+  );
+
+  // AppDelegate.{h,m}
+  const iosRoot = path.resolve(projectRoot, 'ios', targetName);
+
+  fs.copyFileSync(
+    path.resolve(templateRoot, 'ios/AppDelegate.h'),
+    path.resolve(iosRoot, 'AppDelegate.h')
+  );
+
+  fs.copyFileSync(
+    path.resolve(templateRoot, 'ios/AppDelegate.m'),
+    path.resolve(iosRoot, 'AppDelegate.m')
+  );
+
+  // Podfile
+  const podfileRoot = path.resolve(projectRoot, 'ios/Podfile');
+  fs.copyFileSync(path.resolve(templateRoot, 'ios/Podfile'), podfileRoot);
+
+  // update target
+  let podFileStr = fs.readFileSync(podfileRoot, { encoding: 'utf-8' });
+  podFileStr = podFileStr.replace('{{ targetName }}', targetName);
+
+  fs.writeFileSync(path.resolve(projectRoot, 'ios/Podfile'), podFileStr, { encoding: 'utf-8' });
+  if (fs.existsSync(path.resolve(projectRoot, 'react-native.config.js'))) {
+    fs.unlinkSync(path.resolve(projectRoot, 'react-native.config.js'));
+  }
+
+  // Info.plist -> add splash screen
+  IosPlist.modifyAsync(iosRoot, 'Info', (config) => {
+    config['UILaunchStoryboardName'] = 'SplashScreen';
+    return config;
+  });
+
+  fs.copyFileSync(path.resolve(templateRoot, 'App.js'), path.resolve(projectRoot, 'App.js'));
+
+  // 4. yarn + install deps
+  Logger.log('🧶 Yarning...');
+  await spawnAsync('yarn', ['install'], { cwd: projectRoot });
+
+  if (clearCache) {
+    Logger.log('🧶 Clearing cache...');
+
+    const podLockfilePath = path.resolve(projectRoot, 'ios', 'Podfile.lock');
+    if (fs.existsSync(podLockfilePath)) {
+      fs.unlinkSync(podLockfilePath);
+    }
+  }
+
+  if (!platform) {
+    const { selectedPlatform } = await inquirer.prompt({
+      type: 'list',
+      name: 'selectedPlatform',
+      message: 'Which platform are you working on?',
+      choices: [
+        { value: 'ios', name: 'iOS' },
+        { value: 'android', name: 'Android' },
+        { value: 'web', name: 'Web' },
+      ],
+    });
+
+    platform = selectedPlatform;
+  }
+
+  if (platform === 'web') {
+    // TODO
+  } else {
+    await podInstallAsync(path.resolve(projectRoot, 'ios'));
+
+    const command = `run-${platform}`;
+    Logger.log(`🛠  Building for ${platform}...this may take a few minutes`);
+    Logger.log();
+
+    await spawnAsync('react-native', [command, '--no-packager'], {
+      cwd: projectRoot,
+      stdio: 'inherit',
+    });
+
+    const { child: packagerProcess } = spawnAsync('yarn', ['react-native', 'start'], {
+      cwd: projectRoot,
+      stdio: 'inherit',
+    });
+
+    const { child: storiesProcess } = spawnAsync('yarn', ['stories'], {
+      cwd: projectRoot,
+      stdio: 'inherit',
+    });
+
+    process.on('SIGINT', () => {
+      packagerProcess.kill('SIGINT');
+      storiesProcess.kill('SIGINT');
+      process.exit(1);
+    });
+
+    process.on('SIGTERM', () => {
+      packagerProcess.kill('SIGTERM');
+      storiesProcess.kill('SIGTERM');
+      process.exit(1);
+    });
```

Comment:
Let's reuse this fancy pattern from `expo-stories/server/index.ts`, shall we?
```suggestion
    [`exit`, `SIGINT`, `SIGUSR1`, `SIGUSR2`, `uncaughtException`, `SIGTERM`].forEach(eventType => {
      process.on(eventType, () => {
        packagerProcess.kill(eventType);
        storiesProcess.kill(eventType);
        process.exit(1);
      });
    });
```